test/fixtures/users.yml

```diff
@@ -11,5 +11,4 @@ john:
 
 jane:
   email: "jane@example.com"
-  salt: <%= salt = "asdasdastr4325234324sdfds" %>
-  crypted_password: <%= Sorcery::CryptoProviders::BCrypt.encrypt("secret", salt) %>
+  activation_token: "ShfpGexbjiGMpbpXbzLj"
```

Comment:
Can we make the value `secret_token`, that way you know what it is during tests :)